### ISSUE TITLE
Microscopy techniques - new instance (and maybe a clean-up)

### ISSUE THREAD

**Issue by UlrikeS91** [HumanBrainProject/openMINDS_controlledTerms#99]
@ingrre asked if we could add "confocal light-sheet microscopy" and I think that it would make sense to do so. @tgbugs I checked the method ontology but couldn't find it there. Would it make sense to add it? 

Additionally, I'm kind of wondering where we draw the line for the microscopy related terms? Below is a list of all microscopy term that we added so far:

- Brightfield microscopy
- Confocal microscopy
- Dual View Inverted Selective Plane Illumination microscopy
- Electron microscopy
- Epifluorescent microscopy
- Fluorescence microscopy
- Focused Ion Beam Scanning Electron microscopy
- Light microscopy
- Light Sheet Fluorescence microscopy
- Microscopy
- Polarized Light microscopy
- Scanning Electron microscopy
- Serial Section Transmission Electron microscopy
- Transmission Electron microscopy
- Two-Photon Excitation Fluorescence microscopy
- Widefield Fluorescence microscopy
- Widefield microscopy

The easy one first: "microscopy" needs to be removed. This is a duplication since it exists as an experimental approach (which obviously makes most sense anyway). 

Then, I think widefield fluorescence microscopy & epifluorescent microscopy might be synonym (see here: https://oni.bio/nanoimager/super-resolution-microscopy/epifluorescence-microscopy/).  

So, in short, my questions are:
1. Add "confocal light-sheet microscopy" - yes or no?
2. Remove "microscopy" (actual not a question)
3. "widefield fluorescence microscopy" same as "epifluorescent microscopy" - yes or no?
4. If 3. is answers with no, rename "epifluorescent microscopy" to "epifluorescence microscopy" to be consistent with the other fluorescence techniques?

@lzehl feel free to comment as well :)

**Comment by lzehl** [comment]
@UlrikeS91 I agree to removing "microscopy" but maybe next year (if this is already in use in EBRAINS) I don't want to do another mapping right now for migration. But I will check how far this term is in use.

The rest I leave to @tgbugs

**Comment by MaaikevS** [comment]
@UlrikeS91 @lzehl, "microscopy" is not in use so can be removed

**Comment by lzehl** [comment]
@MaaikevS  would you prepare the PR for the removal of microscopy? I will delete it afterwards from the EBRAINS KG

**Comment by MaaikevS** [comment]
@lzehl Sure I can make a PR. Do you mean, remove the file? Or are there some intricate details I am not aware of?

**Comment by lzehl** [comment]
@MaaikevS yes. just remove the jsonld file. the rest has to happen in the EBRAINS KG.

**Comment by MaaikevS** [comment]
Done! See PR #145

**Comment by lzehl** [comment]
Result of discussion with @tgbugs : 

Let's add definitions to see what we might want to merge, but I think besides the widefield fluorescence microscopy & epifluorescent microscopy these can be all considered as different techniques

**Comment by UlrikeS91** [comment]
Thanks to everybody for all the discussion (online and offline) and the PRs to make first fixes! To see where we are with this issue, I'd like to sumnmarize and identify the parts that are still open. 

During the discussion, we pretty much agreed that all of the listed microscopy techniques will remain (except for microscopy which is captured as an experimental approach in openMINDS).

I've copied in the list of techniques from above here again with comments about their status:

(previously) existing technique | definition added? | other notes 
-- | -- | --
Microscopy | - | deleted
Brightfield microscopy | yes | 
Confocal microscopy | yes | 
Electron microscopy | yes | 
Fluorescence microscopy | yes | 
Light microscopy | yes | 
Light Sheet Fluorescence microscopy | yes | 
Polarized Light microscopy | yes | 
Widefield microscopy  | yes | 
Electron tomography | yes | wasn't in my original list
Dual View Inverted Selective Plane Illumination microscopy | no | 
Epifluorescent microscopy | no | this is the same as widefield fluorescence microscopy, delete one and add other as synonym
Focused Ion Beam Scanning Electron microscopy | no | 
Scanning Electron microscopy | no | 
Serial Section Transmission Electron microscopy | no | 
Transmission Electron microscopy | had one already | 
Two-Photon Excitation Fluorescence microscopy | had one already | 
Widefield Fluorescence microscopy | had one already | this is the same as epifluorescence microscopy, delete one and add other as synonym

This means that the ones without a definition should get one and we should also have a look at the ones that already had a definition again just to check that we are happy with them.
All of these should also get ontology IDs if possible, so we could fix those in parallel when adding the definitions and/or check the existing definitions.  

Here the questions when I opened the issue:

> So, in short, my questions are:
> 
> 1. Add "confocal light-sheet microscopy" - yes or no?
> 2. Remove "microscopy" (actual not a question)
> 3. "widefield fluorescence microscopy" same as "epifluorescent microscopy" - yes or no?
> 4. If 3. is answers with no, rename "epifluorescent microscopy" to "epifluorescence microscopy" to be consistent with the other fluorescence techniques?

1. Still needs an answer, but based on the previous discussions, I would say that we should add it unless one of the existing ones would already cover it. I will look into it and either comment again or make a PR for it.
2. done
3. I looked into it and they are the same. I will check which one of the two has been mostly used in the KG and do the mapping if necessary, and of course delete one of them. 
4. We talked about this in a way before and concluded to stick to the term that is used more by the community. I'll check but I assume that it will remain "epifluorescent" (if we use this one and not widefield fl. microscopy). 

To-Do:
- [x] add definitions (and ontology ID, etc.) for 
     - [x] Dual View Inverted Selective Plane Illumination microscopy
     - [x] Focused Ion Beam Scanning Electron microscopy
     - [x] Scanning Electron microscopy
     - [x] Serial Section Transmission Electron microscopy
- [x] review/revise definitions (and add ontology ID, etc.) for 
     - [x] Transmission Electron microscopy
     - [x] Two-Photon Excitation Fluorescence microscopy
- [x] "Widefield Fluorescence microscopy" and "Epifluorescent microscopy"
     - [x] review/revise definition for "Widefield Fluorescence microscopy"
     - [x] merge the terms (based on which is more commonly used) and add other as synonym
     - [x] add "epifluorescence microscopy" as synonym (or new name and add all other ones as synonyms)
     - [x] clean up KG 
- [x] "confocal light-sheet microscopy":
     - [x] check if any other term covers this technique, if not add it as a technique

I think this is it ☺️ Once we fixed the to-do list items, we should be able to close this ticket! I'll work on this today.
